src/components/Title.module.scss

```diff
@@ -0,0 +1,8 @@
+.titleStyle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: xx-large;
+  height: 300px;
```

Comment:
The height should be something like 30% (literally `height: 30%`. So it will not take 2 screens on mobile when you would make it narrower.

Reply:
understood, fixed